scripts/createReleasePipeline/bashScripts/createResources.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+#Create App Service Plan
+az appservice plan create --resource-group $1 \
+    --name $2-plan \
+    --is-linux \
+    --sku B1
+
+#Create Webapp
+az webapp create --resource-group $1 \
+    --plan $2-plan \
+    --name $2 \
+    --runtime "python|2.7"
```

Comment:
why is the runtime python?  is that just because our first app was a python app... does it matter?  is it required?  can it be left out?

Reply:
I seem to remember @d2burkhalter saying we had to put something there and it could not be blank?